searchlib/src/vespa/searchlib/test/features/distance_closeness_fixture.h

```diff
@@ -55,7 +55,16 @@ struct DistanceClosenessFixture : BlueprintFactoryFixture, IndexEnvironmentFixtu
     RankProgram::UP          rankProgram;
     std::vector<TermFieldHandle> fooHandles;
     std::vector<TermFieldHandle> barHandles;
-    DistanceClosenessFixture(size_t fooCnt, size_t barCnt, const Labels &labels, const vespalib::string &featureName);
+    std::shared_ptr<search::tensor::DenseTensorAttribute> attr_tensor;
```

Comment:
Consider renaming member variable to `tensor_attr`.

Reply:
Done